modules/textures/src/lib/parsers/basis-module-loader.ts

```diff
@@ -95,8 +87,8 @@ async function loadBasisEncoder(options) {
   let wasmBinary = null;
 
   [BASIS_ENCODER, wasmBinary] = await Promise.all([
-    await loadLibrary(BASIS_CDN_ENCODER_JS, 'textures', options),
-    await loadLibrary(BASIS_CDN_ENCODER_WASM, 'textures', options)
+    await loadLibrary('basis_encoder.js', 'textures', options),
```

Comment:
Should we follow the pattern we are establishing in Draco and export `BASIS_EXTERNAL_LIBRARIES` namespace etc?

Reply:
DOne